package.json

```diff
@@ -72,10 +70,7 @@
   ],
   "license": "MIT",
   "main": "shrinkpack",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/JamieMason/shrinkpack.git"
-  },
+  "repository": "JamieMason/shrinkpack",
```

Comment:
https://docs.npmjs.com/files/package.json#repository

I was able to check that this holds also for npm@2, https://github.com/npm/npm/blob/v2.13.2/doc/files/package.json.md#repository